src/main/java/gov/cms/mat/cql_elm_translation/controllers/CqlConversionController.java

```diff
@@ -63,7 +65,7 @@ public CqlConversionPayload cqlToElmJson(
             .validateUnits(validateUnits)
             .resultTypes(resultTypes)
             .build();
-
+    cqlParsingService.getCqlBuilderLookups(cqlData, accessToken);
```

Comment:
I'm wondering what is the use of getCqlBuilderLookups here?

Reply:
looks like something that I used for testing from UI.  good catch, thank @sb-cecilialiu!